### ISSUE TITLE
implement Dune query for DAO airdrop

### ISSUE THREAD

**Issue by SCBuergel** [hoprnet/hopr-devrel#212]
Depends on #211 so that we're not just relying on Q to do these sort of things

### DoD
Gnosis Safe CSV Airdrop App compatible CSV file with the airdrop recipients and amounts is available

**Comment by thewanderingeditor** [comment]
@SCBuergel Is this already covered by #149?
